mypy/errorcodes.py

```diff
@@ -160,8 +160,10 @@ def __str__(self) -> str:
     "General",
     default_enabled=False,
 )
-UNSAFE_VARIANCE: Final = ErrorCode("unsafe-variance", "Incorrect usages of variance", "General")
-UNUSED_IGNORE: Final = ErrorCode("unused-ignore", "Ignore comment is unused", "General")
+UNSAFE_VARIANCE: Final[ErrorCode] = ErrorCode(
+    "unsafe-variance", "Incorrect usages of variance", "General"
+)
+UNUSED_IGNORE: Final[ErrorCode] = ErrorCode("unused-ignore", "Ignore comment is unused", "General")
```

Comment:
why explicitly specify these `Final`s?

Reply:
There was some weird bug where it couldn't infer it.

Reply:
i would add a comment/issue link

Reply:
#373